R/run.R

```diff
@@ -64,3 +64,67 @@ for (country in countries) {
     }
   }
 }
+
+
+create_test_case <- function(iso3c) {
+  
+  country <- squire::population$country[squire::population$iso3c==iso3c][1]
+  json <- NULL
+  try({
+    json_path <- file.path("https://raw.githubusercontent.com/mrc-ide/global-lmic-reports/master/",iso3c,"input_params.json")
+    json <- jsonlite::read_json(json_path)
+  })
+  
+  ## convert this into params for deterministic
+  beta_to_R0 <- function(beta, dur_IMild, dur_ICase, prob_hosp, mixing_matrix) {
+    squire:::adjusted_eigen(dur_IMild = dur_IMild, dur_ICase = dur_ICase,
+                            prob_hosp = prob_hosp,
+                            mixing_matrix =  mixing_matrix) * beta
+  }
+  
+  contact_matrix = squire::get_mixing_matrix(country)
+  population = squire::get_population(country)
+  
+  
+  betas <- unlist(lapply(json, "[[", "beta_set"))
```

Comment:
:D jokes

Reply:
fyi, Rich would suggest:

```suggestion
  betas <- vapply(json, "[[", numeric(1), "beta_set")
```

because the lapply and unlist are kinda rogue sometimes. vapply will type and size check your return vector.

Reply:
Hah you have discovered my deep love for lapply. Yup agree